src/data/ToDoList.js

```diff
@@ -0,0 +1,6 @@
+import ToDo from '../models/ToDo';
+export default [
+    new ToDo(Math.random().toString(),"list 1"),
+    new ToDo(Math.random().toString(),"list 2"),
+    new ToDo(Math.random().toString(),"list 3")
+]
```

Comment:
You can see the red icon at the end of the file. It means that eof( End of file ) is missing. Each file should have a empty line at the end.